dashboards/heroku/status.json

```diff
@@ -11,6 +11,7 @@
         "line_label": "%name% %code%",
         "display": "LINE",
         "draw_null_as_zero": false,
+        "format": "number",
```

Comment:
These kinds of changes "breaks" the comparison to the dashboards in the database. This is only if they have never been saved before through the UI, even without further changes.

The UI makes sure fields like `format` are always set. Fields that have some kind of default either in the database or in the front-end graph editor. The initial import and the default storage in the database can be different. These changes make sure it's the same for every dashboard and we have some kind of shared default from now on, so I think we should keep these changes.
However, the system thinks these dashboards are outdated. It will triggers a bunch of inaccurate outdated in-app messages and notifications (if we want to send notifications).

I don't know if we want to fix those kind of differences. If so, we should do that in PR https://github.com/appsignal/appsignal-server/pull/8659.